ci/docker/docker-compose-php74.yml

```diff
@@ -0,0 +1,62 @@
+version: '2.2'
+
+services:
```

Comment:
I would suggest only add config to overide. So it's clear what is overridden. FYI: you can validate the end result with docker config